mysql/datadog_checks/mysql/const.py

```diff
@@ -235,6 +235,9 @@
     'wsrep_flow_control_sent': ('mysql.galera.wsrep_flow_control_sent', MONOTONIC),
     'wsrep_cert_deps_distance': ('mysql.galera.wsrep_cert_deps_distance', GAUGE),
     'wsrep_local_send_queue_avg': ('mysql.galera.wsrep_local_send_queue_avg', GAUGE),
+    'wsrep_replicated_bytes': ('mysql.galera.wsrep_replicated_bytes', GAUGE),
+    'wsrep_received_bytes': ('mysql.galera.wsrep_received_bytes', GAUGE),
```

Comment:
`wsrep_received_bytes` is `Total size of write-sets received from other nodes` (cf Galera doc: https://galeracluster.com/library/documentation/galera-status-variables.html#wsrep-received-bytes)
We should also include `wsrep_received`: `Total number of write-sets received from other nodes`, so that we have their **size** & **number**.
It's useful to understand if an increased traffic (in bytes) is caused by more write-sets or bigger write-sets, so understand the traffic pattern.